app/helpers/notification_count.rb

```diff
@@ -2,12 +2,17 @@ module ExercismWeb
   module Helpers
     module NotificationCount
       def notification_count
-        sql = "SELECT COUNT(note.id) AS tally FROM notifications note INNER JOIN submissions sub ON note.iteration_id=sub.id WHERE note.user_id=#{current_user.id} AND note.read='f'"
+        sql = "SELECT COUNT(note.id) FROM notifications note INNER JOIN submissions sub ON note.iteration_id=sub.id WHERE note.user_id=#{current_user.id} AND note.read='f'"
 
-        notifications_count = ActiveRecord::Base.connection.execute(sql).to_a.first["tally"].to_i
+        notifications_count = ActiveRecord::Base.connection.select_value(sql).to_i
```

Comment:
Have to check this, but I think this achieves the same thing as the previous code.